java/ql/test/library-tests/constants/getStringified.ql

```diff
@@ -0,0 +1,9 @@
+import semmle.code.java.Variable
```

Comment:
Shouldn't this be `import java`, or is there any advantage in importing specifically `Variable` (and the other classes transitively I assume)?
(Though it appears the other older tests in this directory are doing the same.)

Reply:
Yeah, conventionally if whatever you're after is brought in by `import java` then just do that.

Reply:
Done!